deals/deals_test.go

```diff
@@ -21,35 +23,102 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestStoreSimple(t *testing.T) {
-	t.Parallel()
-	dnet, err := ldevnet.New(t, 1)
-	checkErr(t, err)
-	defer dnet.Close()
-
-	m := New(tests.NewTxMapDatastore(), dnet.Client)
+func TestStore(t *testing.T) {
+	numMiners := []int{1}
+	for _, nm := range numMiners {
+		t.Run(fmt.Sprintf("CantMiners%d", nm), storeMultiMiner(t, nm))
+	}
+}
```

Comment:
This tests using our Deals Module to make a deal and check that makes all the process until fully active.
Mostly, all this PR (and #79 ) was to have this working.